hack/common.sh

```diff
@@ -301,6 +297,18 @@ os::build::internal::build_binaries() {
     local host_platform=$(os::build::host_platform)
     local platform
     for platform in "${platforms[@]}"; do
+      os::build::export_targets_and_binaries "${platform}" "$@"
+
+      local -a nonstatics=()
+      local -a tests=()
+      for binary in "${binaries[@]}"; do
+        if [[ "${binary}" =~ ".test"$ ]]; then
```

Comment:
Wow... right you are!  Tricky bash...

I stopped reading at "When it is used, the string to the right of the operator is considered an extended regular expression and matched accordingly (as in regex(3))." and didn't see the bit about quoting special characters.